### PR TITLE
Improve README and developer experience

### PR DESCRIPTION
**Pull request by CommandMC**
This PR makes changes to the README (see commit about it for details), brings VSCode's configuration up to speed again, and resolves a warning (which wasn't an issue in our use case, but it's still nice to not see that every time you run `yarn dev` IMO)

I've tested the Content Security Policy a bit (see notes on commit adding it) and it seems to not introduce any issues, but if you have some time, please verify this :^)

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [X] Tested the feature and it's working on a current and clean install.
- [X] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by Nocccer — CHANGES_REQUESTED** [review]
Pretty good work. Somehow the Patreon picture is not loaded for me.

image

**Review by Nocccer — APPROVED** [review]
Ok second time it loaded...

**Comment by CommandMC** [comment]
The badges very rarely fail to load, that's true. Sadly can't do anything about that in the README, the user will just have to reload

File: README.md [hunk 1]
```diff
@@ -118,137 +138,119 @@ Thanks to the community Heroic was translated to almost 40 different languages s
...
-Download the `heroic_x.x.x_amd64.deb` from the Releases section
+Heroic is available on Flathub, so you should be able to easily install it on most distros with Software Centers (Pop!\_Shop, Discover, etc.)
 
-```bash
-sudo dpkg -i heroic_x.x.x_amd64.deb
-```
-
-#### Debian/Ubuntu (third party `apt` repository)
-
-You can add the [**Raul Dipeas**](https://rauldipeas.surge.sh) _personal_ [`apt` repository](https://github.com/rauldipeas/apt-repository):
```
Comment (flavioislima) [inline README.md:134]: I think some people use this repo, we should keep it.

**Review by flavioislima — APPROVED** [review]
Good stuff! :D

**Pull request merged**